File: csharp/dsf/dsf/src/core/TaskInterruptedException.cs
using System;

namespace com.baidu.dsf.core
{

    /// 
    /// <summary>
    /// @author arksea
    /// </summary>
    public class TaskInterruptedException : Exception
    {

        public TaskInterruptedException(string name, Exception ex) : base(name, ex)
        {
        }

        public TaskInterruptedException(string name) : base(name)
        {
        }
    }

}

File: java/dsframework/src/main/java/dsf/core/TaskInterruptedException.java
package dsf.core;

/**
 *
 * @author arksea
 */
public class TaskInterruptedException extends RuntimeException {

    public TaskInterruptedException(String name, Throwable ex) {
        super(name, ex);
    }

    public TaskInterruptedException(String name) {
        super(name);
    }
}
